angular/src/app/keyword-mapping/keyword-mapping.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { KeywordMapping } from './keyword-mapping.interface';
+
+@Injectable()
+export class KeywordMappingService {
+
+	constructor(private http: HttpClient) { }
+
+	getKeywordMapping(keywordMapping: KeywordMapping){
+		const url = 'http://localhost:5000/keywordMapping';
```

Comment:
use the constant you defined in the env file

Reply:
fixed....